x-pack/solutions/security/plugins/security_solution/public/onboarding/components/onboarding_body/cards/alerts/alerts_card.tsx

```diff
@@ -5,24 +5,41 @@
  * 2.0.
  */
 
-import React, { useCallback, useMemo } from 'react';
+import React, { useCallback, useMemo, useState } from 'react';
 import { EuiFlexGroup, EuiFlexItem, EuiIcon, EuiLink, EuiSpacer } from '@elastic/eui';
 import { SecurityPageName } from '@kbn/security-solution-navigation';
+import { css } from '@emotion/css';
 import { SecuritySolutionLinkButton } from '../../../../../common/components/links';
 import { OnboardingCardId } from '../../../../constants';
 import type { OnboardingCardComponent } from '../../../../types';
-import { OnboardingCardContentImagePanel } from '../common/card_content_image_panel';
+import { OnboardingCardContentAssetPanel } from '../common/card_content_asset_panel';
 import { CardCallOut } from '../common/card_callout';
 import { CardSubduedText } from '../common/card_subdued_text';
-import alertsImageSrc from './images/alerts.png';
 import * as i18n from './translations';
+import type { CardSelectorListItem } from '../common/card_selector_list';
+import { CardSelectorList } from '../common/card_selector_list';
+import { ALERTS_CARD_ITEMS_BY_ID, ALERTS_CARD_ITEMS } from './alerts_card_config';
+import { useOnboardingContext } from '../../../onboarding_context';
+import { DEFAULT_ALERTS_CARD_ITEM_SELECTED } from './constants';
+import { useStoredSelectedCardItemId } from '../../../hooks/use_stored_state';
 
 export const AlertsCard: OnboardingCardComponent = ({
   isCardComplete,
   setExpandedCardId,
   setComplete,
   isCardAvailable,
 }) => {
+  const { spaceId } = useOnboardingContext();
+  const [toggleIdSelected, setStoredSelectedAlertsCardItemId] = useStoredSelectedCardItemId(
+    'alerts',
+    spaceId,
+    DEFAULT_ALERTS_CARD_ITEM_SELECTED.id
+  );
+
+  const [selectedCardItem, setSelectedCardItem] = useState(
+    ALERTS_CARD_ITEMS_BY_ID[toggleIdSelected]
+  );
```

Comment:
Consider using one state instead of two, with a memoized `selectedCardItem`, so we don't need to keep both values in sync manually:

```suggestion
  const [selectedAlertId, setSelectedAlertId] = useStoredSelectedCardItemId(
    'alerts',
    spaceId,
    DEFAULT_ALERTS_CARD_ITEM_SELECTED.id
  );

  const selectedCardItem = useMemo<CardSelectorAssetListItem>(
    () => ALERTS_CARD_ITEMS_BY_ID[selectedAlertId],
    [selectedAlertId]
  );
```

then, down at like 57, we only need to call `setSelectedAlertId`:

```ts
  const onSelectCard = useCallback(
    (item: CardSelectorListItem) => {
      setSelectedAlertId(item.id);
    },
    [setSelectedAlertId]
  );
```